link-grammar/prune.c

```diff
@@ -573,14 +573,14 @@ static inline bool matches_S(connector_table *ct, Connector * c, char dir)
  */
 static int mark_dead_connectors(connector_table *ct, Exp * e, char dir)
 {
-	int count;
-	count = 0;
+	int count = 0;
+	Connector dummy;
+	init_connector(&dummy);
```

Comment:
You don't need a new pull request; just make changes and push to this same branch, and the new changes automatically become part of this pull request.

The compiler, in theory, is supposed to be able to make some of this optimization on its own, but its not clear just how smart the compiler actually is...